abuild.in

```diff
@@ -61,6 +61,28 @@ error() {
 	logcmd "ERROR: $pkgname: $1"
 }
 
+amove() {
+	[ -n "${1##/*}"   ] || return 0
+	[ -n "$subpkgdir" ] || return 0
+	[ -e "$pkgdir/$1" ] || return 0
+
+	cd "$pkgdir"
+	if [ -n "$2" ]; then # find mode
+		mkdir -p "$subpkgdir/$1"
+		find "$1" ! -path "$1" -prune ! -type d -name "$2" \
+			-exec mv '{}' "$subpkgdir/$1/" \;
+	else # file/dir mode
+		mkdir -p "$subpkgdir/${1%/*}"
+		mv "$1" "$subpkgdir/${1%/*}"
+	fi
+
+	# cleanup
+	rmdir -p "$1" || true
+	rmdir -p "${1%/*}" || true
+	cd "$subpkgdir"
```

Comment:
This could probably be done via the dirstack, but it isn't technically [POSIX](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/cd.html). Either way, should be resolved now.